internal/rbd/rbd_attach.go

```diff
@@ -193,27 +196,34 @@ func waitForPath(ctx context.Context, pool, namespace, image string, maxRetries
 	return "", false
 }
 
-// Check if rbd-nbd tools are installed.
-func checkRbdNbdTools() bool {
+// set features available with rbd-nbd, and NBD module loaded status.
+func setRbdNbdToolFeatures() {
 	// check if the module is loaded or compiled in
+	isModuleLoaded := true
```

Comment:
if os.Stat fails with any other error than IsNotExists `isModuleLoaded` remains `true`
`

Reply:
Yes it should be set to true as before. We should just make sure /sys/module/nbd exists.

Reply:
> if os.Stat fails with any other error than IsNotExists isModuleLoaded remains true

Yes, this is intentional.